remote-api/src/main/resources/alfresco/messages/rest-framework-messages.properties

```diff
@@ -15,5 +15,6 @@ framework.exception.DeletedResource=In this version of the REST API resource {0}
 framework.exception.RequestEntityTooLarge=The file can't be uploaded because it's larger than the maximum upload size
 framework.exception.InsufficientStorage=The file upload exceeds the content storage allowance
 framework.exception.JsonpCallbackNotAllowed=For security reasons the callback parameter is not allowed
+framework.exception.InvalidNodeType=Type of provided node is invalid
```

Comment:
It might be useful to know what the node id is, for the situation that multiple nodes are supplied.

Reply:
Good point. I will add the ID.